resources/component_runit_supervisor_upstart.rb

```diff
@@ -0,0 +1,67 @@
+include ComponentRunitSupervisorResourceMixin
+
+provides :component_runit_supervisor, platform_family: 'rhel' do |node|
+  node['platform_version'].to_i == 6
+end
+provides :component_runit_supervisor, platform: 'fedora' do |node|
+  node['platform_version'].to_i <= 14
```

Comment:
We can kill this. If you're running Fedora 14 you should get a horrible error. The thing went EOL years ago